content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "My first-ever KubeCon - KubeCon+CloudNativeCon North America 2023"
+date: 2023-12-05
+slug: My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023
+---
+
+**Author -** Rajdeep Pal, Student, Techno India University
+
+_This post is a guide for beginners to look at huge and overwhelming events like KubeCon._
+
+# Introduction
+
+![KubeCon](https://raw.githubusercontent.com/kubernetes/website/main/content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/pic1.jpg)
```

Comment:
We use the [figure](https://gohugo.io/content-management/shortcodes/#figure) shortcode for images and diagrams; Also, please consider adding suitable captions and alt attributes in the shortcode itself.